pkg/installer/etcdctl.go

```diff
@@ -44,7 +49,11 @@ func endpointsSplitter(endpoints string) string {
 	endpointsSlice := strings.Split(endpoints, ",")
 	httpEndpointsSlice := make([]string, 0)
 	for _, endpoint := range endpointsSlice {
-		httpEndpointsSlice = append(httpEndpointsSlice, fmt.Sprintf("%s%s", "http://", endpoint))
+		endpoint = strings.TrimPrefix(endpoint, httpsPrefix)
+		if !strings.HasPrefix(endpoint, httpPrefix) {
+			endpoint = fmt.Sprintf("%s%s", httpPrefix, endpoint)
+		}
+		httpEndpointsSlice = append(httpEndpointsSlice, endpoint)
```

Comment:
Are we sure that the user is not able to use `https`? Could you please explain this part a bit?

Reply:
etcdctl commands to ETCD from etcd-shell pod only work over `http` and not `https`

![image](https://user-images.githubusercontent.com/41484746/133646748-8cf3bfdd-9141-412e-b077-606f73afbeb1.png)

But then again, this is only the case for the version of ETCD **we** install - if the user is validating against a different ETCD cluster then maybe `https ` is ok and therefore should be allowed?

Reply:
Fix me if i'm wrong but as i see `endpointsSplitter` called for user specified endpoints too. So if they pass https we replace the protocol and possibly breaks thinkgs.
Other question why our etcd is not forced to use https?

Reply:
After discussing with @croomes, we've decided to **not** support TLS for this release. 
However I'm going to work on supporting it in a separate PR, and if I get it done in time, we wil implement - otherwise HTTP only is ok for now.